frontstage/__init__.py

```diff
@@ -1,3 +1,4 @@
+import fakeredis as fakeredis
```

Comment:
Why the `as fakeredis` at the end?

Reply:
Was autocompleted when i added fakeredis will remove it before merging.